web_src/js/components/DashboardRepoList.vue

```diff
@@ -445,3 +447,37 @@ export function initDashboardRepoList() {
 export default sfc; // activate the IDE's Vue plugin
 
 </script>
+<style scoped>
+ul {
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
+}
+
+ul li {
+  padding: 0 10px;
+}
+
+ul li:not(:last-child) {
+  border-bottom: 1px solid var(--color-secondary);
+}
+
+.repo-list-link {
+  padding: 6px 0;
+  gap: 6px;
+}
+
+.repo-list-link .svg {
+  color: var(--color-text-light-2);
+}
+
+.repo-list-icon {
+  min-width: 16px;
+  margin-right: 2px;
+}
+
+.list #privateFilterCheckbox .svg {
```

Comment:
I am pretty sure this is not used, because I have cleaned such strange IDs in previous PRs.

Maybe some other CSS names/styles are also not used now?

Reply:
Removed it. I think at least the remaining styles in this file are all used.

Reply:
There might be more cleanup possible in dashboard.css, but should be something for another PR.